src/content/docs/journey-migration/migrating-infra.mdx

```diff
@@ -0,0 +1,230 @@
+            />
+        </Side>
+        <Side>
+        <Steps>
+            <Step>
+            Select the New Relic account you want to monitor your infrastructure data.  
+            </Step>
+            <Step>
+            Select **macOS** under the **Auto-discovery** section.
+            </Step>
+            <Step>
+            Select **Begin installation**.
+            </Step>
+        </Steps>
+        New Relic will provide a command for you to run on your macOS host. Just copy, paste, and run the command on your host, answer the prompts, and you're done!
+        </Side>
+        </SideBySide>
+        </TabsPageItem>
+        <TabsPageItem id="4">
+        <SideBySide>
+        <Side>
+            <img
+            title="Kubernetes guided install"
+            alt="An image displaying New Relic's guided installation for Kubernetes"
+            src={journeyGuidedK8}
+            />
+        </Side>
+        <Side>
+        <Steps>
+            <Step>
+            Select the New Relic account you want to monitor your infrastructure data.  
+            </Step>
+            <Step>
+            Select **Kubernetes** under the **Auto-discovery** section.
+            </Step>
+            <Step>
+            Select **Begin installation**.
+            </Step>
+        </Steps>
+        The guided install will take you through the Kubernetes implementation process. Follow the prompts and you'll be all set. If you need any help, see our [Kubernetes installation guide](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/#before-start)
+        </Side>
+        </SideBySide>
+        </TabsPageItem>
+        <TabsPageItem id="5">
+        <SideBySide>
+        <Side>
+            <img
+            title="Docker guided install"
+            alt="An image displaying New Relic's guided installation for Docker"
+            src={journeyGuidedDocker}
+            />
+        </Side>
+        <Side>
+        <Steps>
+            <Step>
+            Select the New Relic account you want to monitor your infrastructure data.  
+            </Step>
+            <Step>
+            Select **Docker** under the **Auto-discovery** section.
+            </Step>
+            <Step>
+            Select **Begin installation**.
+            </Step>
+        </Steps>
+        New Relic will provide a command for you to run on your host. Copy, paste, and run the command, and you're done! 
+        </Side>
+        </SideBySide>
+        </TabsPageItem>
+    </TabsPages>
+</Tabs>
+
+New Relic also offers a whole suite of on-host integrations. From **Add Data > Infrastructure & OS**, just select your desired integration from our list to get started. 
+
+## Explore premade dashboards [#add-dashboard]
+
+New Relic provides out-of-the-box dashboards from the moment you set up an infrastructure integration. These dashboards are pre-configured, allowing you to begin visualizing your data with zero set-up time. 
```

Comment:
set-up s/be setup https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/set-up-setup

Reply:
Fixed!